starter-code/css/main.css

```diff
@@ -1,3 +1,46 @@
+/* reset */
+
+/* typography  ...including h1 margins and padding etc */
+
+/* layout */
+    .main {
+        right: 0;
+        left: 0;
+        width: 100%; 
+    }
+
+    .navbar {
+        position: fixed;
+        right: 0;
```

Comment:
Same here! You can remove the right/left: 0. What you have to do is to reset the css (body has paddings by deffault and that's what you got a little white border there)